### ISSUE TITLE
PLIP (WIP) confusing where to delete an image

### ISSUE THREAD

**Issue by djay** [plone/Products.CMFPlone#1434]
# user problem/motivation

(from support request). To reproduce
1. A new user uploaded a new image via tinymce
2. They changed their mind about which image
3. they clicked on an image again which lets them upload a new replacement
4. they are confused where to go to remove the old one they still can see in the image popup

# options

1. improve training/docs so users know to 
   a) look at the breadcrumb inside the image popup 
   b) find this using main breadcrumbs 
   c) click folder contents
   d) change pages until they find the image (if lots of content)
   e) delete the image
2. all remove inside the image popup
3. helper to ask if you want to remove the old one if replacing an image
4. ??

**Comment by djay** [comment]
@plone/ux-team ideas?

**Comment by frisi** [comment]
this is indeed a usability problem.
i like the idea of having a delete-button (X icon on hover) or a context menu in the image dialog.

another idea would be a button "open this folder" (or something like that) that takes the user to the folder_contents view of the currently displayed folder. this would safe us from duplicating functionality (like the delete within the tiny-image-popup and also offer users more possibility (mass delete, rename, etc)

**Comment by hvelarde** [comment]
yes, this is a problem that has no obvious solution right now (perhaps uploading all images to a single folder?); in the future, when all content types are containers, obviously the uploaded image will be inside the object and you can just go to folder contents to manage that; that's the same approach we used in [collective.nitf](https://github.com/collective/collective.nitf), for instance.

**Comment by djay** [comment]
updated with your suggestions. @hvelarde and @frisi . Can you check to make sure I got them right? If not feel free to change them.

**Comment by tkimnguyen** [comment]
One of the customizations we have made is to upload images and files to one specific folder in the site. It simplifies the UX nicely.

**Comment by tisto** [comment]
@djay no activity since 2016. Closing this PLIP.
